frontend/src/components/IstioStatus/IstioStatus.tsx

```diff
@@ -220,7 +214,7 @@ export const IstioStatusComponent: React.FC<Props> = (props: Props) => {
     }, true);
   };
 
-  const tooltipPosition = props.location === MASTHEAD ? TooltipPosition.bottom : TooltipPosition.top;
+  const tooltipPosition = pathname === MASTHEAD ? TooltipPosition.bottom : TooltipPosition.top;
```

Comment:
```suggestion
  const tooltipPosition = TooltipPosition.bottom;
```
The pathname will never be `MASTHEAD` because that was just a made-up token. We can get just get rid of the `MASTHEAD` constant completely and just assume that this is in the masthead and set the position to bottom.

Reply:
right, an odd logic, removed